### DISCUSSION TITLE
Delete Progress Sync Account

### DISCUSSION THREAD

**Discussion by Rorschach1010** [koreader/koreader#10522] [Q&A]
I wanted to try KOreaders own sync server so I tried to register an account. However, seems like the name I wanted to use was already used by someone. This is no problem at all and I just tried another name which worked and I'm now logged in.

This led me to the question, how can we delete accounts? I can't find an option for it so I'm guessing that the sync server contains lots of dead accounts? You also can't change the name/password?

Obviously, I don't want to delete my account as I intend to use it. It's just a general question. Maybe it's best to implement some basic account management or at least auto-prune accounts without activity for 2 (?) years?

**Comment by Frenzie** [comment]
I suppose that should be possible if someone wants to do that.
https://github.com/koreader/koreader-sync-server/blob/10e89da70318ea9ff4199b1dfc3140017454e55f/app/controllers/1/syncs_controller.lua#L153-L161

**Comment by Rorschach1010** [comment]
That's actually interesting. Would be cool if someone who knows how to do it would look into it.
